forestserviceprototype/specialuseform/views.py

```diff
@@ -15,3 +16,11 @@ def submit(request):
 
 def submitted_permit(request):
     return render(request, "specialuseform/submitted_permit.html")
+
+# @todo: Fix redirect so users don't get stuck in admin screen. See #7.
+@login_required(login_url='/admin/')
+def applications(request):
+    permits = Permit.objects.all()
```

Comment:
If you want the permits in a particular order, consider using `.order_by()` on this queryset: https://docs.djangoproject.com/en/1.10/ref/models/querysets/#django.db.models.query.QuerySet.order_by.